viral/cache_2p_sessions.py

```
@@ -635,9 +647,7 @@ def process_session(
     wheel_freeze = get_wheel_freeze(session_sync) if wheel_blocked else None
     trials = add_imaging_info_to_trials(trials, session_sync, wheel_freeze)
 
-    with open(
-        HERE.parent / "data" / "cached_2p" / f"{mouse_name}_{date}.json", "w"
-    ) as f:
+    with open(CACHE_PATH / f"{mouse_name}_{date}.json", "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)